lib/recurly.js

```diff
@@ -274,20 +311,21 @@ export class Recurly extends Emitter {
    * @private
    */
   parent () {
-    let reset = false;
-
-    if (this.hostedFields) {
-      // Check integrity of hostedFields. If fields are dead or do not match chosen selectors, reset
-      if (this.readyState > 1 && !this.hostedFields.integrityCheck(this.config.fields)) reset = true;
-    }
+    // Check integrity of hostedFields. If fields are dead or
+    // do not match chosen selectors, reset
+    let reset = (
```

Comment:
_nit_ `reset` to me sounds like a function. Maybe `shouldReset` would be more appropriate?